skbio/stats/distance/tests/test_base.py

```diff
@@ -467,6 +468,32 @@ def test_to_data_frame_default_ids(self):
                            index=['0', '1'], columns=['0', '1'])
         assert_data_frame_almost_equal(df, exp)
 
+    def test_to_series_1x1(self):
+        series = self.dm_1x1.to_series()
+        exp = pd.Series([0.0],
+                        pd.MultiIndex(levels=[['a'], ['a']],
+                        labels=[[0], [0]]))
+        exp = exp[exp > 0]
+        assert_series_almost_equal(series, exp)
+
+    def test_to_series_3x3(self):
```

Comment:
Can you change this test to use a 4x4 matrix, e.g. the one I defined [here](https://github.com/biocore/scikit-bio/pull/1406/files#r73383377)? A matrix larger than 3x3 is useful here to test the ordering of ID pairs in the series.

Can you also add a test case for a 2x2 matrix? That way we'll have test cases for 1x1 (empty series), 2x2 (single value series), and 4x4 (multi-valued series).